### PR TITLE
Project skeleton

### PR DESCRIPTION
**Pull request by notlfish**
### What?
Pull request to set up the project's skeleton

### Before merging
The following tasks are to be finished before merging this pull request.
- [x] Add redux directory following the ducks pattern with empty an empty `ConfigureStore.js` file and empty duck files for books and categories.
- [x] Add the following React components (mostly empty for the time being):
  - [x] App
    - [x] Navbar
    - [x] Bookstore
      - [x] BooksList
        - [x] BookItem
      - [x] AddBookForm
    - [x] Categories

### After review:
The following changes were implemented after the first code review:
- [x] Add a hardcoded (temporary) list of books as the state of the `Bookstore` component.
- [x] Handle the logic to render these books from `Bookstore` to `BookList`, and from there to `BookItem`.
- [x] Render the HTML for a book, following the [given design](https://github.com/microverseinc/curriculum-react-redux/blob/main/bookstore/images/bookstore.png).
- [x] Add author input field to the `add new book` form.

### CODE REVIEW

File: src/components/BooksList.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import BookItem from './BookItem';
+
+const BooksList = (props) => {
```
Comment (eliasecasta) [inline src/components/BooksList.js:10]: - [x] Your BooksList component must render a list of books using the Book component, you are handling it and using it correctly but you have to actually render a book list with sample books for this project.

File: src/components/BookItem.js [hunk 2]
```diff
@@ -0,0 +1,7 @@
+import React from 'react';
+
+const BookItem = () => (
+  <p>Under Construction</p>
```
Comment (eliasecasta) [inline src/components/BookItem.js]: - [x] Your book item component should be the one creating each book, receiving a book prop, and rendering the book title and author, please change your logic accordingly to match the project requirements.

File: src/components/BookItem.js [hunk 3]
```diff
@@ -0,0 +1,7 @@
+import React from 'react';
+
+const BookItem = () => (
```
Comment (eliasecasta) [inline src/components/BookItem.js]: - [x] This component should also add a delete book button to each book, please add it to comply with the project requirements. (It doesn't have to work right now, just has to be present)

File: src/components/AddBookForm.js [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+      <select placeholder="Category">
+        <option value="Fiction">Fiction</option>
+        <option value="Fantasy">Fantasy</option>
+        <option value="Magic Realism">Magic Realism</option>
+        <option value="Essay">Essay</option>
+      </select>
+      <button type="submit">Add book </button>
+    </form>
+  </div>
+);
```
Comment (eliasecasta) [inline src/components/AddBookForm.js]: - [x] You should also have an input to get the book's author, please add it to comply with the project requirements.

File: src/components/BookItem.js (hunk 2 above)
Comment (eliasecasta) [inline src/components/BookItem.js]: - [x] Your BookItem.js should not display the "Under Construction" text, that should be placed in your Categories component, please do so to comply with the project requirements.

**Review by eliasecasta — CHANGES_REQUESTED** [review]
# Status:  Changes Required ♻️
_Check the comments under the review_
<hr>

Hello @notlfish, I am Elias, your reviewer for this PR

Well done so far with your project!

Almost everything is set up correctly, great job so far!

I will mention to you a small summary of things that need to be changed in order to keep moving:

- [x] Please check the comments below.

Please correct those issues and send another review request, good luck, and well done so far! 

Code review by Elias Castañeda

_(Feel free to leave any questions or comments in the PR thread if something is not 100% clear)_

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

As described in the [Code reviews limit Policy](https://microverse.zendesk.com/hc/en-us/restricted?return_to=https%3A%2F%2Fmicroverse.zendesk.com%2Fhc%2Fen-us%2Farticles%2F1500004088561) you have a limited number of code reviews per project (you can check the exact number in your Dashboard). If you think that the code review was not fair, you can [request a second opinion using this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by GzimAsani — APPROVED** [review]
# Hello @notlfish           

## Thank you for implementing all the project requirements asked previously, you've done a great job :trophy:  . Your project is **ACCEPTED :green_circle:** and I wish you the best of luck in your next projects :+1: 

 ## Optional suggestions
 _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
 
 You can also consider:
 
- [  **_Optional_** ] Your delete button is not being appended to the page. Kindly add that.

 Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

 **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
 
 _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**